app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+class CommandError(Exception):
+    """Checks command input for right format."""
+
+
+def check_command(func: callable) -> callable:
+    def wrapper(command: str) -> callable:
+        if not command.startswith("cp") or len(command.split(" ")) < 3:
```

Comment:
What if the command  `cp1`  ? Will it work correctly?

Reply:
thanks for comment
added space 

def check_command(func: callable) -> callable:
    def wrapper(command: str) -> callable:
        if not command.startswith("cp ") or len(command.split(" ")) < 3:
            raise CommandError("Please enter valid command")
        return func(command)

Reply:
you didn't fix it. You need to explicitly check  command to ensure it is exactly "cp"

Reply:
updated with checking the first item in string split:

 if command.split(" ")[0] != "cp" or len(command.split(" ")) < 3: